.github/workflows/hugo-build.yml

```diff
@@ -6,10 +6,6 @@ on:
   pull_request:
     branches:
       - master
-  schedule:
```

Comment:
This has been replaced by [scheduled-daily](https://github.com/pulumi/docs/blob/master/.github/workflows/sheduled-daily.yml).